JamGame/Assets/Scripts/Level/Config/DayAction.cs

```diff
@@ -152,6 +152,16 @@ public void Execute(Executor executor)
     [Serializable]
     public class LoseGame : IDayAction
     {
+        [ReadOnly]
+        [SerializeField]
+        private string loseCause = "No Cause";
```

Comment:
```suggestion
        private string cause = "No Cause";
```

Reply:
Also why not to requre `cause` in constructor? I suppose `"No Cause"` is invalid anyway